journalbeat/magefile.go

```diff
@@ -22,20 +22,42 @@ package main
 import (
 	"context"
 	"fmt"
+	"strings"
 	"time"
 
 	"github.com/magefile/mage/mg"
 	"github.com/magefile/mage/sh"
+	"github.com/pkg/errors"
 
 	"github.com/elastic/beats/dev-tools/mage"
 )
 
 func init() {
 	mage.BeatDescription = "Journalbeat ships systemd journal entries to Elasticsearch or Logstash."
 
-	// TODO filter platforms
+	mage.Platforms = mage.Platforms.Filter("linux")
 }
 
+var (
+	deps = map[string]func() error{
+		"linux/386":      installLinux386,
+		"linux/amd64":    installDefaultLinux,
+		"linux/arm64":    installLinuxArm64,
```

Comment:
Namewise I think that all the architecture acronyms should be in all caps, like `installLinuxARM64`.